doc/developer/feature-benchmark.md

```diff
@@ -12,6 +12,12 @@ First, make sure your python environment is up to speed:
 bin/pyactivate --dev
 ```
 
+The `--help` option can be used to show supported options:
+
+```
+./mzcompose run feature-benchmark --help
+```
+
 To run all benchmarks:
```

Comment:
nit: this is no longer all scenarios is it? the big ones are excluded by default now